the-telephone-app-fe/src/components/JoinRoom/JoinRoom.js

```diff
@@ -104,7 +111,8 @@ const JoinRoom = () => {
             <h4 className="jr-left-player">PLAYERS</h4>
             <div className="jr-user">
               <span className="jr-choice-number">
-                <select name="" id="">
+                {role == 1? (
+                  <select onChange={handleSelectChange} name="" id="">
```

Comment:
Select tag and option tags inside should have different indents.
Also fix the player number of the option of value 20.